mtp_api/apps/credit/management/commands/create_prisoner_credit_notices.py

```diff
@@ -97,10 +92,21 @@ def handle(self, path, prison, date=None, **options):
         bundle = PrisonerCreditNoticeBundle(prison.name, prisoners, date)
         bundle.render(str(path))
 
-    def get_housing(self, prisoner_number):
-        if not can_access_nomis():
-            return
-        try:
-            return nomis_get_location(prisoner_number, retries=2)['housing_location']
-        except (TypeError, KeyError, ValueError, requests.RequestException):
-            return
+
+def parsed_date_or_yesterday(date) -> datetime.date:
+    if date:
+        date = parse_date(date)
+        if not date:
+            raise CommandError('Date cannot be parsed, use YYYY-MM-DD format', {'date_string': date})
+    else:
+        date = now().date() - datetime.timedelta(days=1)
```

Comment:
I know this function's implementation is extracted straight as it was, but wouldn't be better to use the usual `django.utils.timezone.localdate()` instead of `now().date()`?

Reply:
yeah we might as well!